dutree/dutree.py

```diff
@@ -304,10 +305,38 @@ class DuScan:
     "Disk Usage Tree scanner"
 
     def __init__(self, pathname):
+        self._dev_allow = []
+        self._dev_deny = []
+        self._dev = lstat(pathname).st_dev
         self._path = self._normpath(pathname)
         self._tree = None
         self._check_path()
 
+    def skip_other_filesystems(self):
+        "add self._dev to self._dev_allow"
+        self._dev_allow = [self._dev]
+
+    def skip_proc_sys_filesystems(self):
+        "add proc and sys devices to self._dev_deny"
+
+        try:
+            proc_device = lstat('/proc').st_dev
+        except Exception:
+            warnings.warn('/proc not found, so device is not skipped',
+                          OsWarning)
```

Comment:
Yes. I did not mention this, but I (we) prefer the 4 space indent over the line-up indent.

That is:
```python
            warnings.warn('/proc not found, so device is not skipped',
                          OsWarning)
```
|
v
```python
            warnings.warn(
                '/proc not found, so device is not skipped', OsWarning)
```
This starts making sense once you get more/longer arguments, which we'll see below:
```python
    parser = argparse.ArgumentParser(prog='dutree',                            
                                     description='dutree shows a summary of \  
                                        the directories/files which take up\   
                                        the most space.')
```
|
v
```python
    parser = argparse.ArgumentParser(
        prog='dutree', description=(
            'dutree shows a summary of the directories/files which take up '   
            'the most space.'))
```
or, if you prefer:
```python
    parser = argparse.ArgumentParser(prog='dutree', description=(
        'dutree shows a summary of the directories/files which take up '
        'the most space.'))
```